### ISSUE TITLE
Small text size on MobileSafari

### ISSUE THREAD

**Issue by edurur** [Carpetsmoker/hello-css#6]
I absolutely love this CSS template. But when I test webpage with MobileSafari text size appears relatively small. See attached screenshots for reference.

MobileSafari default:

<a href="https://ibb.co/VM1bXMC"><img src="https://i.ibb.co/gt2QstW/DE8-CFF2-F-6-D3-B-4-DCC-8-E0-A-91045555-CBC2.png" alt="DE8-CFF2-F-6-D3-B-4-DCC-8-E0-A-91045555-CBC2" border="0"></a>

MobileSafari with reader mode activated:

<a href="https://ibb.co/sQjbb0B"><img src="https://i.ibb.co/xGhXXp4/A840-F802-94-A9-488-A-8-A55-D6634-ECF20-CB.png" alt="A840-F802-94-A9-488-A-8-A55-D6634-ECF20-CB" border="0"></a>

**Comment by arp242** [comment]
Which device do you have? It scales down some text on mobile depending on screen size, but maybe it's a bit too much. I find it looks kind of okay on my iPhone SE (but that's probably not the most common size).

**Comment by edurur** [comment]
I got iPhone 6S. It got .7” more screen compared to SE.

**Comment by edurur** [comment]
https://github.com/arp242/hello-css/blob/7356f40e82375015644eecd954b877a06d0fcb9a/dist/base.min.css#L37

I just tested with Chrome DevTools, changing 12px to 18px seems okay. But its gonna break other media sizes, should fix that too.

**Comment by arp242** [comment]
18px seems rather huge to me. The default desktop size is 16px. Generally you want to make the fonts a bit smaller on mobile devices, not larger.

I changed it from 12px to 14px on my site (arp242.net), and at a glance that seems about right to me on my iPhone SE? Need to test some more later (maybe I should increase line-height a wee bit, too).

**Comment by edurur** [comment]
> 18px seems rather huge to me. The default desktop size is 16px. Generally you want to make the fonts a bit smaller on mobile devices, not larger.

Yeah, I looked desktop size later, 18px seems okay for me but not for the sake of consistency.

![Check](https://user-images.githubusercontent.com/28399740/63574389-572c6300-c590-11e9-83ce-d16488fa0b69.jpeg) this low-quality picture for real world comparison. iPhone 6S 4.7", iPhone 5S (same screen size as your SE) 4", Samsung Wave 3.3" (ancient phone with ancient browser)

Also 14px is better and usable, 12px was way too small, I guess you should change size for all other media sizes.

**Comment by robjbrain** [comment]
I'm having a similar issue. It seems like for some reason the width of the page is defaulting to around 8-900px where as on other sites it would be the actual width of the browser.

Here's an example comparison with Wikipedia.

I haven't figured out why yet.

<img width="538" alt="image" src="https://user-images.githubusercontent.com/10143910/185793862-1ae712a8-3c5f-4795-b45e-0166c1a5ce9a.png">

<img width="519" alt="image" src="https://user-images.githubusercontent.com/10143910/185793883-a2716a22-6017-4872-b3f1-502095200454.png">
